app/controllers/users_controller.rb

```diff
@@ -113,6 +113,18 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def test_mail
+    user = find_resource(:edit_users)
+    begin
+      MailNotification[:tester].deliver(:user => user, :email => params[:user_email])
+    rescue Net::SMTPAuthenticationError, Net::SMTPServerBusy, Net::SMTPSyntaxError, Net::SMTPFatalError, Net::SMTPUnknownError, Timeout::Error,
+           Errno::ECONNRESET, Errno::ECONNABORTED, Errno::EPIPE, Errno::ETIMEDOUT => e
+      Foreman::Logging.exception("Unable to deliver email", e)
+      render :json => e.message, :status => :unprocessable_entity and return
```

Comment:
I'd go for returning a message like "sending email has failed, check logs for detailed info" instead of the e.message